cmd/harmony/main.go

```diff
@@ -492,7 +533,7 @@ func main() {
 	}
 
 	utils.Logger().Info().
-		Str("BlsPubKey", hex.EncodeToString(nodeConfig.ConsensusPubKey.Serialize())).
+		Str("BlsPubKey", "").
```

Comment:
change to empty string? can just remove it

Reply:
Added actual keys.

Reply:
Removed it. Actual keys caused seg fault.